lib/3dview/workers/litegcodeviewer.js

```diff
@@ -1,6 +1,6 @@
 self.addEventListener('message', function(e) {
   console.log("New message received by worker", e.data.data.length)
-  importScripts("/lib/threejs/three.min.js");
```

Comment:
Will this cause problems for the web version?

Reply:
When I test the web version using vs code live server I don't see any.